app/Http/Middleware/ConvertStringBooleans.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\TransformsRequest;
+
+class ConvertStringBooleans extends TransformsRequest
+{
+    protected function transform($key, $value)
+    {
+        if($value === 'true' || $value === 'TRUE')
+            return true;
+
+        if($value === 'false' || $value === 'FALSE')
+            return false;
+
+        return $value;
+    }
+}
```

Comment:
Eloquent can cast model properties to native PHP types - is this what you need?

Reply:
This is for inputs